### PR TITLE
Update VMs and pick up VS2019 16.10

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Comment by Neumann-A** [comment]
Requires: #18320 due to https://bugreports.qt.io/browse/QTBUG-94038. (or otherwise qtbase will fail.)

**Comment by BillyONeal** [comment]
> Requires: #18320 due to https://bugreports.qt.io/browse/QTBUG-94038. (or otherwise qtbase will fail.)

I don't believe we build that with clang here?

**Comment by Neumann-A** [comment]
> I don't believe we build that with clang here?

I could have sworn I have seen the exact same error after updating to VS 16.10 with cl (and not clang-cl which had a totally different error in qsimd.h). Unfortunately, it was in my vcpkg as a submodule folder which I had to play around to make my manifest/custom triplet work as I wanted to. TL;DR the log is of course gone since I knew how to work around that issue by updating to 6.1.1 (or at least that was what I thought).

Lets just wait and see if CI bubbles up that error or not.

**Comment by Neumann-A** [comment]
Also discord link of another person in the include qt channel:
https://discord.com/channels/400588936151433218/435701082962919424/848669161336078336

File: ports/libgpg-error/portfile.cmake [hunk 1]
```diff
@@ -53,20 +53,21 @@ if(VCPKG_TARGET_IS_WINDOWS)
...
-    
+
     set(exec_prefix "\${prefix}")
     set(libdir "\${prefix}/lib")
     set(includedir "\${prefix}/../include")
     set(GPG_ERROR_CONFIG_LIBS "-L\${libdir} -lgpg-errord")
     configure_file("${SOURCE_PATH}/src/gpg-error.pc.in" "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/gpg-error.pc" @ONLY)
     vcpkg_fixup_pkgconfig()
     vcpkg_copy_pdbs()
+    file(REMOVE "${CURRENT_PACKAGES_DIR}/lib/COPYING.LIB;${CURRENT_PACKAGES_DIR}/debug/lib/COPYING.LIB")
```
Comment (Neumann-A) [inline ports/libgpg-error/portfile.cmake:70]: Isn't this missing the copyright know in the windows path?

Comment (BillyONeal) [reply ports/libgpg-error/portfile.cmake:70]: No, we were already copying COPYING.LIB as share/copyright (via `LICENSE_SUBPATH COPYING.LIB`)

Comment (BillyONeal) [reply ports/libgpg-error/portfile.cmake:70]: ( See https://twitter.com/MalwareMinigun/status/1404548271373111297/photo/1 -- share/copyright is present, those were duplicates )

**Comment by JackBoosY** [comment]
Can you please also add `libgtk-3-dev` to apt list?
See https://github.com/microsoft/vcpkg/pull/17111.
Since wxwidgets only support gtk 3, we can't change this dependency to vcpkg internal.

**Comment by Neumann-A** [comment]
I wonder if CMake should be updated to 3.20.4 first. I saw that they are a bunch of commits updating the xml table for VS new vs VS old.

**Comment by BillyONeal** [comment]
> Can you please also add `libgtk-3-dev` to apt list?
> See #17111.
> Since wxwidgets only support gtk 3, we can't change this dependency to vcpkg internal.

If I have to redeploy the Linux VMs again I'll add.

**Comment by talregev** [comment]
For gtk3 we can create a gtk3 and gtk4 ports similar to opencv ports.

**Pull request merged**